airflow/providers/amazon/aws/log/cloudwatch_task_handler.py

```diff
@@ -97,17 +107,12 @@ def get_cloudwatch_logs(self, stream_name: str) -> str:
         :param stream_name: name of the Cloudwatch log stream to get all logs from
         :return: string of all logs from the given log stream
         """
-        try:
-            events = self.hook.get_log_events(
-                log_group=self.log_group,
-                log_stream_name=stream_name,
-                start_from_head=True,
-            )
-            return "\n".join(self._event_to_str(event) for event in events)
-        except Exception:
```

Comment:
I am propagating the AWS error message to the task log. I think this will be useful for debugging IAM/connection issues.